app/models/book.rb

```diff
@@ -1,6 +1,6 @@
 class Book < ApplicationRecord
-  has_many :reviews
-  has_many :books_by_author
+  has_many :reviews, dependent: :destroy
+  has_many :books_by_author, dependent: :destroy
```

Comment:
What does this do?

Reply:
It destroys resources that are dependent on the thing being destroyed!

Reply:
Sweet! Much easier.